src/helpers.ts

```diff
@@ -198,6 +206,37 @@ export function loadOrCreateReceipt(
   return receipt;
 }
 
+/**
+ * helper function that returns the value of a bid in a given slot index, in Wei.
+ * used by the RAM Minter.
+ * @param basePrice Base price (or reserve price) of the auction in Wei, string representation
+ * @param slotIndex Slot index to query
+ * @returns slotBidValue Value of a bid in the slot, in Wei
+ */
+export function slotIndexToBidValue(
+  basePrice: string,
```

Comment:
nit: any reason to not just have this input type be a BigInt?

Reply:
Good catch - updated this to BigInt 🙏